manifests/install/charts/as-a-second-scheduler/templates/configmap.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: v1
```

Comment:
The config hard-codes the setup as "enable coscheduler only", so a regular user may want an option to customize the config when installing the Helm chart. Is that possible to parameterize it in Helm chart?

Reply:
I can add a value list like `enabled-plugins` to `values.yaml` to let user choose which plugin to enable in following PRs.